packages/dnb-eufemia/src/extensions/payment-card/icons/index.js

```diff
@@ -16,6 +16,7 @@ import SagaGold from './SagaGold'
 import SagaPlatinum from './SagaPlatinum'
 import VisaDefault from './VisaDefault'
 import VisaPlatinum from './VisaPlatinum'
+import Hourglass from './Hourglass'
```

Comment:
```suggestion
import { hourglass as Hourglass } from '../../../icons'
```

btw. should this be a medium (24px) icon?

Reply:
this hourglass icon import suggested looks fine so I will change it to use this